Code/GraphMol/RascalMCES/rascal_cluster.cpp

```diff
@@ -0,0 +1,141 @@
+//
```

Comment:
I guess you added this for testing?
If you think it has value going forward then maybe it's worth adding a comment up here indicating what this is for